doc/releases/changelog-dev.md

```diff
@@ -91,6 +91,9 @@
 * CI will now fail if coverage data fails to upload to codecov. Previously, it would silently pass
   and the codecov check itself would never execute.
   [(#5101)](https://github.com/PennyLaneAI/pennylane/pull/5101)
+  
+* The `FABLE` template is added for efficient block encoding of matrices. Users can now call FABLE to efficiently construct circuits according to a user-set approximation level. 
```

Comment:
Awesome, don't forget to add your name to the authors list at the very bottom of this file (in alphabetical order)